<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNSS_DOC_OPTIMIZERDockWidgetBase</class>
 <widget class="QDockWidget" name="GNSS_DOC_OPTIMIZERDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>498</height>
   </rect>
  </property>
  <property name="whatsThis">
   <string comment="Add REGINA station" extracomment="Add REGINA station"/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="accessibleDescription">
   <string comment="Add REGINA station"/>
  </property>
  <property name="windowTitle">
   <string>GNSS_DOC</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="Ground_network">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>71</width>
      <height>401</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string extracomment="Change stations' framework"/>
    </property>
    <property name="title">
     <string>Grd_Netwrk_Util</string>
    </property>
    <widget class="QPushButton" name="Add_station">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>53</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string comment="Add REGINA station" extracomment="Add REGINA station"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/Add.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Remove_station">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/delet_station.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Move_station">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/move.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Import_stations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>52</width>
       <height>52</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/import.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="export_view">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>51</width>
       <height>53</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/export.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Refresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/Refresh.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="glonass">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Constel</string>
    </property>
    <widget class="QPushButton" name="gps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/usa.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="galileo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/europ.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/russia.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="beidou">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/plugins/GNSS_DOC_OPTIMIZER/icons/china.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <connections/>
</ui>
